### PR TITLE
UIIN-2667: Jest/RTL: Cover ImportRecord component with unit tests

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
https://folio-org.atlassian.net/browse/UIIN-2667
Here was added jest tests for `ImportRecord` component
Screenshot 2024-04-25 at 05 41 30

### CODE REVIEW

File: src/components/ImportRecord/ImportRecord.test.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { createMemoryHistory } from 'history';
+import { Router } from 'react-router';
+import ky from 'ky';
+import { waitFor } from '@folio/jest-config-stripes/testing-library/react';
+import ImportRecord from './ImportRecord';
+import buildStripes from '../../../test/jest/__mock__/stripesCore.mock';
+import '../../../test/jest/__mock__';
```
Comment (artem-blazhko) [inline src/components/ImportRecord/ImportRecord.test.js:8]: This import is redundant

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,12 +13,11 @@
 * Do not remove field from the form when its value is an empty string. Fixes UIIN-2787.
 * Update Permission name for Inventory: Set records for deletion and staff suppress. Refs UIIN-2855.
 * ECS Member tenant - add `Shared` and `Held by` facets for the `Classification` browse. Refs UIIN-2813.
-* Jest/RTL: Cover CalloutRenderer component with unit tests. Refs UIIN-2665.
+* Jest/RTL: Cover ImportRecord component with unit test. Refs UIIN-2667.
 
 ## [11.0.3] (IN PROGRESS)
 
 * Fetch facets with the same query as `Contributor` records when the search is done by selecting browse result. Refs UIIN-2859.
-* Change the `Effective location` call numbers facet query to not see locations assigned to items with an empty `Effective call number` field. Refs UIIN-2871.
```
Comment (artem-blazhko) [inline CHANGELOG.md:21]: Do you really need to remove previously added information in CHANGEOG file?

**Review by vashjs — APPROVED** [review]

**Review by artem-blazhko — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (vashjs) [reply CHANGELOG.md]: Fixed

File: src/components/ImportRecord/ImportRecord.test.js (hunk 1 above)
Comment (vashjs) [reply src/components/ImportRecord/ImportRecord.test.js]: fixed

**Pull request merged**